### PR TITLE
Refactored the Code and Changes the function Name

### LINKED ISSUE #3: Refactor the codebase
* Refactor the codebase

### LINKED ISSUE #5: The syntax and naming conventions is not upto standards
**Actual Behaviour**
* The syntax and naming conventions is not upto standards

**Expected Behaviour**
* Meaningful variable naming
* Make the syntax, indentation correct

### PR DESCRIPTION
**Pull request by rkishan516**
Fixes #3
Fixes #5
Fixes #10 

Changes: [Refactored the Code and Changes the functions and variables Name.]

### CODE REVIEW

**Comment by rkishan516** [comment]
@Akshat-Jain Please look at it.

**Comment by rkishan516** [comment]
@Akshat-Jain I have sent the PR to him but He is unable to resolve the issue so this is the final PR from my side for both.

**Comment by rkishan516** [comment]
@Akshat-Jain I have done this PR 9 days ago till now no response.

File: js/main.css [hunk 1]
```diff
@@ -27,12 +23,12 @@ body {
 img {
 	float:left;
 	display: inline-block;
-	  width:15px;
-	  height: 15px;
-	  padding-right: 5px;
-	  background-repeat: no-repeat;
-	  background-position: center center;
-	  background-size: cover;
+	width:15px;
```
Comment (akshatnitd) [inline js/main.css]: Please use consistent spacing here.

File: popup.js [hunk 2]
```diff
@@ -6,35 +6,28 @@ var save_btns=[];
...
-var x= function() 
-{
-    chrome.windows.getAll(function(win_all)
-    {
-        win_all.map(function(win)
-        {
+var store_tab = () => {
+    chrome.windows.getAll( (win_all) => {
+        win_all.map( (win) => {
             b=[];
```
Comment (akshatnitd) [inline popup.js]: Please name the variables like a, b appropiately.

**Review by akshatnitd — CHANGES_REQUESTED** [review]

**Comment by rkishan516** [comment]
@Akshat-Jain Please look once again

**Comment by rkishan516** [comment]
@akshatnitd Please look at it.

File: js/main.css (hunk 1 above)
Comment (akshatnitd) [reply js/main.css]: The spacing is not consistent. It should be `width: 15px;`

**Comment by rkishan516** [comment]
@akshatnitd please now check.

**Comment by rkishan516** [comment]
@akshatnitd Please check it.

**Review by akshatnitd — APPROVED** [review]
Looks good :+1:

**Pull request merged**